### ISSUE TITLE
[Accessibility][Screen reader-M365 Compliance-Advanced eDiscovery-Processing: On navigating on the bar 'Error by File type' Focus initially lands at last bar series and the position of bar series incorrectly numbered.

### ISSUE THREAD

**Issue by VivianSi233** [highcharts/highcharts#17219]
#### Expected behaviour
On navigating on the bar 'Error by File type' focus should initially land at first bar series 'File is malformed' and the position of bar series should be numbered correctly.
For ex: Errors by file type. High charts interactive chart.  region clickable File is malformed., bar series 1 of 7 with 1 bar.  region    1. File is malformed., 93. File is malformed..  graphic Interactive chart graphic Created with Highcharts 9.2.2

#### Actual behaviour
On navigating on the bar 'Error by File type' focus initially lands at last bar series 'File size is zero' inorder to navigate to next bar series user need to press 'left arrow key and navigate in reverse direction and the position of bar series incorrectly numbered.
For ex: Errors by file type. High charts interactive chart.  region clickable File size is zero., bar series 1 of 7 with 1 bar.  region 1. File size is zero., 3. File size is zero..  graphic Interactive chart graphic  Created with Highcharts 9.2.2.
![image](https://user-images.githubusercontent.com/89379019/165420663-a67b26d5-cd7d-41ad-9c7e-61e2175214ab.png)

**Test Environment:**
Test URL: [testcaseforadddatasource - Microsoft 365 compliance](https://df.compliance.microsoft.com/advancedediscovery/cases/v2/50fa33d4-b099-4b0c-a475-8e0c8ba4681a?casename=testcaseforadddatasource&casesworkbench=JobsTab&tid=596ade3a-1abe-4c5b-b7d5-a169c4b05d4a)
Test Account: [admin@ediscoSdf.onmicrosoft.com ](mailto:admin@ediscoSdf.onmicrosoft.com)
OS: Windows 11
Version: 21H2 (OS Build 22000.466)
Browser version: 99.0.1135.5 (Official build) dev (64-bit)
Screen Reader: NVDA

**Repro step:**
1. Open the above URL and login with valid credentials
2. Navigate to the 'Processing' tab and activate it.
3. Further select 'Index status' in 'Views' dropdown and navigate to 'Error by File type' and observe the screen reader announcement.

#### Product version
Highcharts 9.2.2.

#### Affected browser(s)
Microsoft Edge

**Comment by oysteinmoseng** [comment]
Hi @VivianSi233, we are not able to access the test environment you are linking to, as we do not have a password for the account. Are you able to reproduce the chart in jsFiddle or Codepen? This would also help us see the chart configuration code.

**Comment by VivianSi233** [comment]
>
[admin@ediscoSdf.onmicrosoft.com](mailto:admin@ediscoSdf.onmicrosoft.com)
Please try this PW: Pass@word1
